notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb

```diff
@@ -768,7 +670,7 @@
       "source": [
         "#### [Optional] Create FeatureGroup/Features\n",
         "\n",
-        "Create a FeatureGroup pointing to the created BigQuery view for the demo. We will then create features for each column we would like to register.\n"
+        "Create a FeatureGroup pointing to the created BigQuery view for the demo. You will then create features for each column you would like to register.\n"
```

Comment:
Change:
- " You will then create features for each column you would like to register"
to:
- " You then create features for each column you'd like to register"

Reply:
Did the required changes